packages/hmr/lib/initializers/initClient.ts

```diff
@@ -1,7 +1,7 @@
-import { DO_UPDATE, DONE_UPDATE, LOCAL_RELOAD_SOCKET_URL } from '../constant';
-import MessageInterpreter from '../interpreter';
+import { DO_UPDATE, DONE_UPDATE, LOCAL_RELOAD_SOCKET_URL } from '../consts.js';
+import MessageInterpreter from '../interpreter/index.js';
```

Comment:
```suggestion
import { DO_UPDATE, DONE_UPDATE, LOCAL_RELOAD_SOCKET_URL } from '../consts';
import MessageInterpreter from '../interpreter';
```

Reply:
![image](https://github.com/user-attachments/assets/059cb166-f47b-4778-8250-a6caf95692f7)

It need to be placed.
I was trying to solve it on September i've spent sooooooo much time on in, to create it more pretty, but i can't it's not possible because of the tsconfig, which need to look like current, because on other setting everything doesn't want to build